tests/test_service.py

```diff
@@ -28,12 +27,10 @@
 }
 
 
-@contextlib.contextmanager
-def mock_get_rule_file(rule_file):
-    with patch('requests.get') as get:
-        with open(rule_file, 'r') as f:
-            get.return_value.text = f.read()
-        yield
+def read_file(filename):
```

Comment:
`open(filename).read()` also achieves the same thing :)

But given the usage of this function in your tests, it would be helpful if it took a list of paths:
```python
def read_file(*paths):
  full_path = os.path.join(*paths)
  return open(full_path).read()

# Usage example
read_file(test_data_dir, 'mts-test-for-no-match.yaml')
```

Reply:
I really like the idea of the way passing arguments.